Containerfile

```diff
@@ -15,12 +17,11 @@ RUN pip3 install --no-cache-dir \
     netaddr \
     jmespath \
     paramiko \
+    PyYAML \
     ncclient
 
 # Copy application files to eco-ci-cd folder
-COPY . ./eco-ci-cd
-
-WORKDIR /eco-ci-cd
+COPY . .
```

Comment:
Are you sure it's going to work? Is eco-ci-cd default directory?

Reply:
when you set the workdir first the dot is the workdir 

also I've tested it - it works without an issue